app/src/main/res/values/strings.xml

```diff
@@ -58,6 +58,9 @@
     <string name="top_bar_tab_notifications">Notifications</string>
 
     <string name="no_browser">There is no app to install Awala</string>
+    <string name="no_app_to_share">There is no app to share your id</string>
```

Comment:
What conditions must be true for this scenario to happen? I know nothing about this part of Android. :) 

I'm just trying to understand how likely/frequent this can be and what alternatives there are to displaying this message.

Reply:
I forgot to ask about texts in Slack... This is the case where a user clicks on "Share id" and there is no app which is ready to handle this intent (seems unrealistic, but this could happen).

Reply:
Cool, thanks 👍🏾

I think we should involve Maribel here. Presumably we could also not show the button if that's the case?

Reply:
No, the better option is to try send this intent and catch an exception if there is no app which can handle an intent. We can add this logic as you described, but it'll be constant checks using `PackageManager` which is not cool in terms of performance, and it'll be useful for 0.000001% of users :)

Reply:
I see! Can you please share a screenshot of what this looks like on Slack so Maribel can see it?

Reply:
It's just a toast message https://www.google.com/search?newwindow=1&sca_esv=564302744&sxsrf=AB5stBgJAhu0K3attENbiXnUY1-2EqZ92w:1694424800122&q=toast+android&tbm=isch&source=lnms&sa=X&ved=2ahUKEwij8ryioKKBAxVORPEDHTbbCzoQ0pQJegQIDBAB&biw=1510&bih=795&dpr=2#imgrc=j_ueeHKeUnQwaM